testing/mocks/generic.go

```diff
@@ -15,18 +15,32 @@
 package mocks
 
 import (
+	"io"
+	"time"
+
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/cadence"
+	"github.com/onflow/cadence/encoding/json"
+	"github.com/onflow/cadence/runtime/tests/utils"
 	"github.com/onflow/flow-go/ledger"
 	"github.com/onflow/flow-go/ledger/common/hash"
 	"github.com/onflow/flow-go/ledger/complete/mtrie/node"
+	"github.com/onflow/flow-go/ledger/complete/mtrie/trie"
 	"github.com/onflow/flow-go/model/flow"
 
 	"github.com/optakt/flow-dps/models/dps"
+	"github.com/optakt/flow-dps/rosetta/identifier"
+	"github.com/optakt/flow-dps/rosetta/object"
 )
 
 // Global variables that can be used for testing. They are non-nil valid values for the types commonly needed
 // test DPS components.
 var (
-	GenericHeight = uint64(42)
+	GenericLogger = zerolog.New(io.Discard)
+
+	GenericHeight    = uint64(42)
+	GenericByteSlice = []byte(`test`)
```

Comment:
The convention that I know is to just use `bytes` for a byte slice, so `GenericBytes` would do.